src/backend/tests/unit/utils/test_user_auth.py

```
@@ -0,0 +1,302 @@
+    async def test_get_current_user_whitespace_token_causes_jwt_error(self, mock_db):
+        """Test that whitespace-only token causes JWT validation error."""
+        token = "   "
+        query_param = "valid-api-key"
+        header_param = None
+
+        # Whitespace token will be treated as a JWT token and fail validation
+        with patch("langflow.services.auth.utils.get_current_user_by_jwt") as mock_jwt_auth:
+            # The JWT library will raise a ValueError for malformed tokens
+            mock_jwt_auth.side_effect = ValueError("not enough values to unpack (expected 2, got 1)")
+
+            with pytest.raises(ValueError) as exc_info:
```

Ruff (0.11.9)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)